project.clj

```diff
@@ -95,7 +95,7 @@
              :ezbake {:dependencies ^:replace [[puppetlabs/puppet-server ~ps-version]
                                                [puppetlabs/trapperkeeper-webserver-jetty9 ~tk-jetty-version]
                                                [org.clojure/tools.nrepl "0.2.3"]]
-                      :plugins [[puppetlabs/lein-ezbake "0.2.3"]]
+                      :plugins [[puppetlabs/lein-ezbake "0.3.0-SNAPSHOT"]]
```

Comment:
Will running from this SNAPSHOT work - at least temporarily - in CI?  On the CI node that runs ezbake will lein-ezbake have already been built from latest master and installed into a local .m2/repository so that the dependency can be satisfied?